### ISSUE TITLE
Perinephric infiltration

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11229]
**Preferred term label:**
Perinephric infiltration

**Synonyms**
thickening; hairy kidney

**Definition (free text, please give PubMed ID)**
Thickening of the tissue surrounding the kidney

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Erdheim-Chester Disease (Non Langerhans' cell histiocytosia)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001336
- label: Hairy kidney sign
- parents: [Abnormal renal morphology (HP:0012210)](https://hpo.jax.org/app/browse/term/HP:0012210)
- pmid: PMID:29728208
- def: A radiological sign related to infiltration of histiocytes into the kidneys and perirenal fat. This infiltration, along with thickening of fibrous bands within the perirenal fat, creates a hairy appearance on cross-sectional imaging studies.
- comment: Erdheim-Chester disease is a non-Langerhans cell histiocytosis characterized by the infiltration of foamy histiocytes with fibrosis that can affect many organs, most commonly long bones. Renal involvement commonly manifests in this disease as a histiocytic infiltration into the perirenal fat.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11229
- synonyms: Perinephric infiltration(NONE)
